jetty-util/src/main/java/org/eclipse/jetty/util/component/DumpableCollection.java

```diff
@@ -34,6 +35,12 @@ public DumpableCollection(String name,Collection<?> collection)
         _name=name;
         _collection=collection;
     }
+    
+    public DumpableCollection(String name,Object... item)
```

Comment:
This new constructor is never used. Seems a leftover from some previous refactoring, so I would remove it.